sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/ControlFlowGraph/ControlFlowGraphTest.cs

```diff
@@ -1578,18 +1578,20 @@ public void Cfg_ConditionalAccess()
         public void Cfg_ConditionalAccessNested()
```

Comment:
could you add some CFG tests with conditional access `?` followed
- by `??` operator e.g. `if (aObj?.booleanVal ?? false)`
- by `is` operator
- `(a?.booleanVal == null ? true : false)`

This will help document the shape of the CFG. If these tests already exist, the better